R/check_data.R

```diff
@@ -0,0 +1,131 @@
+#' Check data sources
+#' @description  Check that data are up to date and returning correctly
+#' @return A tibble of latest dates for all sources and data checks for sub-national data
+#' @importFrom tibble tibble
+#' @importFrom purrr map
+#' @importFrom dplyr group_by filter bind_rows summarise pull rename
+
+check_data_sources <- function(){
+  
+# Get data ----------------------------------------------------------------
+  # Get names of countries included in package
+  regions <- dir("R")
```

Comment:
suggest make this a pass in to enable use outside of the working directory of the package. Can also use system.file to find files in the installed R package.